bot.py

```diff
@@ -1714,6 +1712,41 @@ async def before_autoupdate(self):
             self.autoupdate.cancel()
             return
 
+    @tasks.loop()
+    async def log_expiry(self):
+        log_expire_after = self.config.get("log_expiration")
+        if log_expire_after == isodate.Duration():
+            return self.log_expiry.stop()
+
+        now = datetime.utcnow()
```

Comment:
```suggestion
        now = discord.utils.utcnow()
```

For consistency with the rest of the codebase, we use discord.py's `utcnow()`, which includes the timezone info.